app/views/landing_page/_icon.erb

```diff
@@ -95,6 +95,15 @@
 <% when "transfer-arrows" %>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g fill="none" stroke="none" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><path d="M16 1.5l3 3-3 3M19 4.5H.5M16 17.5l3 3-3 3M19 20.5H.5"/><g><path d="M8 9.5l-3 3 3 3M5 12.5h18.5"/></g></g></svg>
 
+<% when "world-flight" %>
+ <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><path d="M23.334 13.962C22.402 19.38 17.682 23.5 12 23.5 5.648 23.5.5 18.352.5 12 .5 5.65 5.648.5 12 .5c3.93 0 7.398 1.97 9.473 4.978"/><path d="M14.794.843l-4.7 2.657.918 1.625-2 3.375-5 1.5 2 6.5 5.5 2V21M20.5 4.255L18.512 6.5"/><path d="M11.72 12.38l-2.656-1.187-2.004 1.154 4.56 2.944 4.544-1.79-1.926 5 2.273-1.182 2.772-5.062 3.456-1.522c.656-.29.94-1.055.65-1.694-.28-.624-1.004-.916-1.64-.662l-10.03 4.003z"/><path d="M18.07 9.847L14 7.297 11.966 8.35l3.696 2.458"/></g></svg>
```

Comment:
I don't know if it's absolutely necessary, but usually I've set the stroke to none (`stroke="#000"` to `stroke="none"`)